src/controllers/auth/login.js

```diff
@@ -124,9 +126,13 @@ define(["apphost", "appSettings", "dom", "connectionManager", "loading", "layout
             html += "</div>";
             html += "</div>";
             html += "</button>";
-        }
+        });
 
-        context.querySelector("#divUsers").innerHTML = html;
+        const divUsers = context.querySelector("#divUsers");
```

Comment:
This file is not transpiled yet.

Reply:
Everything is transpiled :)

The package,json list is for module conversion only (ie turning the import/export into define/return). But Babel runs on everything.

Reply:
OK then :p